ColiseuPlayer/ColiseuPlayer.swift

```diff
@@ -294,13 +307,8 @@ public class ColiseuPlayer: NSObject
             return
         }
 
-        do {
-            if let path = song.path {
-                self.audioPlayer = try AVAudioPlayer(contentsOf: path)
-            }
-        }
-        catch let error {
-            print("AVAudioPlayer error occurred:\n \(error)")
+        if let path = song.path {
+            self.audioPlayer = self.engine.initAudioPlayer(url: path)
```

Comment:
I see this like a Builder and not an Engine, agree? The class is building AudioPlayer's.

Reply:
I see this more like `self.builder.createAudioPlayer`.

Reply:
Yeah, you are correct...